stock_move_backdating/models/stock_move.py

```diff
@@ -21,5 +21,11 @@ def action_done(self):
                 move.date = operation.operation_id.date
                 if move.quant_ids:
                     move.quant_ids.sudo().write({'in_date': move.date})
-
+        # set date_done as the youngest date among the pack operations
+        operations = self.mapped(
+            'linked_move_operation_ids.operation_id')
+        dates = operations.mapped('date')
+        pickings = operations.mapped('picking_id')
+        pickings = pickings.filtered(lambda r: r.state == 'done')
+        pickings.write({'date_done': max(dates)})

```

Comment:
@eLBati, considerato che il transfer di un picking viene fatto uno solo alla volta, assumerei che in self siano presenti solo le move del picking che si sta trasferendo e che in pickings sia presente solo quel picking.

Solo se fosse installato il modulo **stock_picking_wave** potrei processare più picking alla volta ma anche in questo caso verrebbe sempre chiamata l'action_done sull'insieme di move corrispondenti allo stesso picking: https://github.com/odoo/odoo/blob/8.0/addons/stock/stock.py#L947

Mi sfugge forse qualcosa?